definitions/classnames_v2.x.x/flow_>=v0.23.x/classnames_v2.x.x.js

```diff
@@ -1,5 +1,8 @@
+
+type Classes = string | [class: string]: bool } | Array<string>
```

Comment:
Ah -- So in declaration files, anything outside a `declare module {}` is considered global (this is how you can define actually-expected globals to Flow)